rstudio/rstudio
# Open source doc in new column

### Intent

Allow user to open a document in a new column, rather than columns always opening with a blank doc. This is based on #7920. 

### Approach

I added a new command `openSourceDocNewColumn` that mimics the existing `openSourceDoc` command. When opening the file chooser the command creates a new column and sets it to the active column, when the user selects a file the process is the same as with `openSourceDoc` where the file is opened in the active column. If the user cancels out of the file chooser, the new column is removed. 

I looked into what it would take to open files in a new columns from the Files Pane and it ends up being more complicated as it revolves around open file events which differ between file types. I implemented being able to `Shift+Click` a file to open it in a new column on a separate branch but ultimately decided that feature is too risky for this to make it into 1.4.

I also included some general code clean up (removed unused functions and parameters, fixed spacing)
### QA Notes

You should be able to open any file type in a new column, but there should only ever be three columns open at a time. An error will be displayed if the user attempts to open a fourth column. 

![image](https://user-images.githubusercontent.com/5323711/95359302-fc3ef380-0897-11eb-9ea0-e61ca8dfa4f1.png)
